torch/_torch_docs.py

```diff
@@ -8292,25 +8292,30 @@ def merge_dicts(*dicts):
 svd(input, some=True, compute_uv=True, *, out=None) -> (Tensor, Tensor, Tensor)
 
 Computes the singular value decomposition of either a matrix or batch of
-matrices :attr:`input`. The singular value decomposition is represented as a
-namedtuple `(U, S, V)`, such that :attr:`input` `= U diag(S) Vᴴ`.
+matrices :attr:`input`.
+
+The singular value decomposition is represented as a
+namedtuple `(U, S, V)`, such that `input = U diag(S) Vᴴ`.
 where `Vᴴ` is the transpose of `V` for real inputs,
 and the conjugate transpose of `V` for complex inputs.
-If :attr:`input` is a batch of matrices, then `U`, `S`, and `V` are also
+If :attr:`input` is a batch of matrices, then `U`, `S`, `V` are also
 batched with the same batch dimensions as :attr:`input`.
 
+The singular values are returned in descending order. If :attr:`input` is a batch of matrices,
+the singular values of each matrix in the batch are returned in descending order.
+
 If :attr:`some` is `True` (default), the method returns the reduced singular
-value decomposition. In this case, if the last two dimensions of :attr:`input` are
-`m` and `n`, then the returned `U` and `V` matrices will contain only
-`min(n, m)` orthonormal columns.
+value decomposition. In this case, if :attr:`input` is of size `(*, m, n)`,
+the returned `U`, `V` will be of size `(*, m, min(m, n))` and `(*, n, min(m, n))`
+respectively.
 
 If :attr:`compute_uv` is `False`, the returned `U` and `V` will be
 zero-filled matrices of shape `(m, m)` and `(n, n)`
 respectively, and the same device as :attr:`input`. The argument :attr:`some`
 has no effect when :attr:`compute_uv` is `False`.
 
-Supports :attr:`input` of float, double, cfloat and cdouble data types.
-The dtypes of `U` and `V` are the same as :attr:`input`'s. `S` will
+Supports :attr:`input` of float, double, cfloat and cdouble dtypes.
+The dtype of `U` and `V` is the same as that of :attr:`input`. `S` will
```

Comment:
"of \`U\` and \`V\` is" -> "of the output \`U\` and \`V\` tensors will be"

"\`S\` will" -> "The output tensor \`S\` will"

Reply:
Corrected and made uniform through all the functions that use this construction